packages/core/content-manager/admin/src/services/documents.ts

```diff
@@ -166,7 +166,7 @@ const documentApi = contentManagerApi.injectEndpoints({
       }
     >({
       query: ({ model, params }) => ({
-        url: `/content-manager/collection-types/${model}${params ? `?${params}` : ''}`,
+        url: `/content-manager/collection-types/${model}`,
```

Comment:
Can you confirm the original fix still works ?

Reply:
yes i tested this manually